### PR TITLE
cmake,make: speed up man page generation

### PR DESCRIPTION
**Pull request by tchaikov**
and this also address the failure of
```
Exception occurred:
  File "/usr/lib/python2.7/dist-packages/sphinx/writers/manpage.py", line 422, in unknown_visit
    raise NotImplementedError('Unknown node: ' + node.__class__.__name__)
NotImplementedError: Unknown node: ditaa
The full traceback has been saved in /tmp/user/1000/sphinx-err-r65gwE.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```

seems the ditaa plugin is not enabled when rendering manpages.

### CODE REVIEW

File: admin/build-doc [hunk 1]
```diff
@@ -20,7 +20,6 @@ if command -v dpkg >/dev/null; then
       graphviz
       ant
       cython
-      librbd-dev
```
Comment (tchaikov) [inline admin/build-doc]: this partially reverts the change in #9147. @flaf could you help review it? thanks.

**Comment by flaf** [comment]
Ok, I will check in my trustry and Jessie VMs after work. ;)

**Comment by flaf** [comment]
I have tested this repository https://github.com/tchaikov/ceph.git branch `wip-cmake-doc` commit id 0fa22f0d1b163de4125a302fc62b50b1fcdc50db with Trusty and Jessie. I have run `./admin/build-doc`, installed the suggested packages and run again  `./admin/build-doc`. The builds were successful.

**Comment by tchaikov** [comment]
@flaf thanks for testing.

**Comment by flaf** [comment]
But you are welcome. ;)

**Pull request merged**